src/custom/pages/Swap/index.tsx

```diff
@@ -34,7 +34,6 @@ import TradeGp from 'state/swap/TradeGp'
 import { useUSDCValue } from 'hooks/useUSDCPrice'
 import { computeTradePriceBreakdown, FEE_TOOLTIP_MSG } from 'components/swap/TradeSummary/TradeSummaryMod'
 import { useExpertModeManager, useUserSlippageToleranceWithDefault } from 'state/user/hooks'
-import { V2_SWAP_DEFAULT_SLIPPAGE } from 'hooks/useSwapSlippageTolerance'
 import { RowReceivedAfterSlippage, RowSlippage } from '@src/custom/components/swap/TradeSummary'
```

Comment:
not related but path..

Reply:
Exactly. Not related.

Reply:
Just kidding -> https://github.com/gnosis/cowswap/pull/1282/commits/a8f3339ee4bb20e4cd0eaec546300d2bb282b9d7